src/pawns.cpp

```diff
@@ -61,6 +61,14 @@ namespace {
   // Unsupported pawn penalty
   const Score UnsupportedPawnPenalty = S(20, 10);
 
+  // Center Bind Bonus: Two pawns controlling the same central square
+  const Bitboard CenterBindMask[] = {
+    (FileDBB | FileEBB) & (Rank5BB | Rank6BB | Rank7BB),
+    (FileDBB | FileEBB) & (Rank4BB | Rank3BB | Rank2BB)
+  };
+
+  const Score CenterBindBonus = S(16, 0);
```

Comment:
Here we don't add bonus suffix, so i would call it:

const Score CenterBind = S(16, 0);